src/renderer/components/+apps-releases/release.store.ts

```diff
@@ -78,9 +78,16 @@ export class ReleaseStore extends ItemStore<HelmRelease> {
   }
 
   async loadItems(namespaces: string[]) {
-    return Promise
-      .all(namespaces.map(namespace => helmReleasesApi.list(namespace)))
-      .then(items => items.flat());
+    const isLoadingAll = namespaceStore.allowedNamespaces.every(ns => namespaces.includes(ns));
+    const noAccessibleNamespaces = namespaceStore.context.cluster.accessibleNamespaces.length === 0;
```

Comment:
Logic duplication from `KubeObjectStore.loadItems()`.. Maybe extract to upper level? (e.g. `ItemStore`) or reuse somehow in other way..